bootstrap/test/nerves_new_test.exs

```diff
@@ -68,4 +68,13 @@ defmodule Nerves.BootstrapTest do
     end
   end
 
+  test "new project default cookie set", context do
+    in_tmp context.test, fn ->
+      Mix.Tasks.Nerves.New.run([@app_name])
+      assert_file "#{@app_name}/rel/vm.args", fn file ->
+        assert file =~ ~r/.+?(?:-setcookie ).{64}\n/s
```

Comment:
Ah, right. I was thinking the Base64 charset was `[a-zA-Z0-9]`, but it turns out that's only 62 characters. 😅 
So you could do `[a-zA-Z0-9+/]{64}` or `\S{64}` (for 64 non-whitespace characters).